polling_stations/apps/file_uploads/tests/test_check_pending.py

```diff
@@ -0,0 +1,69 @@
+from django.core import mail
+from django.conf import settings
+from django.test import TestCase
+from file_uploads.models import Upload
+from councils.models import Council
+from file_uploads.management.commands import check_pending
+from freezegun import freeze_time
+
+
+class TestCheckPending(TestCase):
+    def setUp(self):
+        self.council1 = Council.objects.create(
+            council_id="E09000001",
+            name="City of London Corporation",
+        )
+        self.council2 = Council.objects.create(
+            council_id="E09000002",
+            name="Westminster City Council",
+        )
+
+    @freeze_time("2021-04-01 13:00:00+00:00")
+    # Find pending uploads lasting 20 minutes or more, email us and log the email as sent
+    def test_handle_long_pending_upload(self):
+        upload1 = Upload.objects.create(
+            gss=self.council1,
+            election_date="2021-05-06",
+            timestamp="2021-04-01 11:45:00+00:00",
+        )
+        upload2 = Upload.objects.create(
+            gss=self.council2,
+            election_date="2021-05-06",
+            timestamp="2021-04-01 11:46:00+00:00",
+        )
+        self.assertEqual(len(Upload.objects.all()), 2)
+        self.assertEqual(Upload.objects.all()[0].warning_about_pending_sent, False)
+        self.assertEqual(Upload.objects.all()[1].warning_about_pending_sent, False)
+        qs = check_pending.Command().handle()
+        self.assertEqual(len(qs), 2)
+        self.assertEqual(len(mail.outbox), 2)
+        self.assertEqual(mail.outbox[0].subject, "File upload failed")
+        self.assertEqual(mail.outbox[1].subject, "File upload failed")
+        self.assertEqual(
+            mail.outbox[0].body,
+            f"File upload failure: {upload1}. Please investigate further.",
+        )
+        self.assertEqual(
+            mail.outbox[1].body,
+            f"File upload failure: {upload2}. Please investigate further.",
```

Comment:
I would hard code these strings - this is why there wasn't a failure when the method was using `self.__str__` - if you hard code the expected string and someone changes the `__str__` method then they'll know what's currently expected.